Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM golang:1.11-alpine3.7 as builder
+
+RUN apk add --no-cache --virtual .build-deps gcc musl-dev make git expat
+RUN apk add vips-dev --verbose --update-cache --repository https://dl-3.alpinelinux.org/alpine/edge/testing/ --repository https://dl-3.alpinelinux.org/alpine/edge/main
+
+ADD . /src
+WORKDIR /src
+RUN make dep
+RUN make test
+RUN make build
+
+#################
+FROM alpine:3.7
+ARG SERVER_ADDR=:8080
+ARG STORE_FILE_ORIGINALS=/images/originals
+ARG STORE_FILE_CACHE=/images/cache
+ARG STORE_FILE_THUMBNAILS=/images/thumbnails
+ARG STORE_S3_REGION=""
+ARG STORE_S3_BUCKET=""
+
+LABEL default.server.addr=${SERVER_ADDR} \
+      default.store.file.originals=${STORE_FILE_ORIGINALS} \
+      default.store.file.cache=${STORE_FILE_CACHE} \
+      default.store.file.thumbnails=${STORE_FILE_THUMBNAILS}
+    
+
+ENV IMAGERESIZER_SERVER_ADDR=${SERVER_ADDR}
+ENV IMAGERESIZER_STORE_FILE_ORIGINALS=${STORE_FILE_ORIGINALS}
+ENV IMAGERESIZER_STORE_FILE_CACHE=${STORE_FILE_CACHE}
+ENV IMAGERESIZER_STORE_FILE_THUMBNAILS=${STORE_FILE_THUMBNAILS}
+ENV IMAGERESIZER_STORE_S3_REGION=${STORE_S3_REGION}
+ENV IMAGERESIZER_STORE_S3_BUCKET=${STORE_S3_BUCKET}
+
```

Comment:
I don't think the ones concerning S3 are needed.

Reply:
Ok